doc/tutorials/platformer/app/web/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
```

Comment:
You only need this file in the `web` directory, have a look at the `web` directory for Klondike.